### DISCUSSION TITLE
使用Xelatex可以正常编译，但是使用LaTexmk就会报错？

### DISCUSSION THREAD

**Discussion by whitemountain-blackwater** [TheNetAdmin/zjuthesis#240] [General]
![image](https://user-images.githubusercontent.com/59988132/160592200-37804a47-a9ef-4d0d-94dd-118be8e01825.png)

**Comment by RobinEllison** [comment]
你这应该提供更详细的信息啊，比如你的系统信息，所用的版本，以及输出的log文档，要不然别人根本帮不到你。
